Vic2ToHoI4/Source/HOI4World/HoI4World.cpp

```diff
@@ -774,25 +776,52 @@ void HoI4::World::addUnrecognizedNations(Mappers::CountryMapper::Factory& countr
 		nation->addTag(nationTag, *names, *hoi4Localisations);
 		countries.emplace(nationTag, nation);
 
+		std::map<std::string, int> popTotalsByCulture;
 		for (const auto& stateId: nation->getCoreStates())
 		{
 			if (auto state = modifiableStates.find(stateId); state != modifiableStates.end())
 			{
 				state->second.addCores({nationTag});
 				state->second.setOwner(nationTag);
 				nation->addState(state->second);
+
+				for (const auto& hoi4ProvinceNumber: state->second.getProvinces())
+				{
+					for (const auto& vic2ProvinceNumber: provinceMapper.getHoI4ToVic2ProvinceMapping(hoi4ProvinceNumber))
+					{
+						const auto& vic2Province = sourceWorld.getProvince(vic2ProvinceNumber);
+						if (!vic2Province)
+						{
+							continue;
+						}
+
+						for (const auto& pop: (*vic2Province)->getPops())
+						{
+							auto [itr, success] = popTotalsByCulture.emplace(pop.getCulture(), pop.getSize());
+							if (!success)
```

Comment:
if (!success)?

Reply:
If it couldn't be emplaced (because there's already an instance of the culture). It means popTotalsByCulture only needs to be searched for the culture once.